api/spec/smithy/model/operations/DescribeCluster.smithy

```diff
@@ -63,4 +63,6 @@ structure DescribeClusterResponse {
     headNode: EC2Instance,
     @documentation("Reason of the failure when the stack is in CREATE_FAILED, UPDATE_FAILED or DELETE_FAILED status.")
     failureReason: String
```

Comment:
question: 
should we remove this? 
Is there a reason why you did not remove it?

(removing this should remove the field from all the other generated files)